doc/manual/src/language/index.md

```diff
@@ -4,9 +4,7 @@ The Nix language is
 
```

Comment:
I am off by one line thanks to github :/
```suggestion
This Nix language conveniently create and compose precise descriptions of how contents of existing files are used to derive new files.
It is:

```
I think the sentence you added works much better as an intro, than as the description of "domain-specific".